### ISSUE TITLE
CTests failing on Orion and Hercules

### ISSUE THREAD

**Issue by DavidNew-NOAA** [NOAA-EMC/GDASApp#1406]
The following GDASApp CTests fail on ```Orion```:
```
test_gdasapp_snow_apply_jediincr
test_gdasapp_snow_letkfoi_snowda
atm_jjob*
```
Output: 
```
/work2/noaa/da/role-da/CI/orion/GDASApp/workflow/PR/1401/global-workflow/sorc/gdas.cd/build/log.ctest
```

```Hercules```:
```
test_gdasapp_util_ioda_example
test_gdasapp_snow_apply_jediincr
test_gdasapp_snow_letkfoi_snowda
test_gdasapp_convert_bufr_adpsfc_snow
atm_jjob*
```
Output:
``` 
/work2/noaa/da/role-da/CI/hercules/GDASApp/workflow/PR/1401/global-workflow/sorc/gdas.cd/build/log.ctest
```

**Comment by DavidNew-NOAA** [comment]
cc @RussTreadon-NOAA @CoryMartin-NOAA

**Comment by CoryMartin-NOAA** [comment]
@DavidNew-NOAA I see the error looks like:
srun: error: Unable to allocate resources: Invalid account or account/partition combination specified

Are any of the following defined in the runtime environment?
```
export SLURM_ACCOUNT=da-cpu
export SALLOC_ACCOUNT=$SLURM_ACCOUNT
export SBATCH_ACCOUNT=$SLURM_ACCOUNT
export SLURM_QOS=debug
```
If `srun` is executed, it looks for some of these by default

**Comment by DavidNew-NOAA** [comment]
@CoryMartin-NOAA Yeah, see https://github.com/NOAA-EMC/GDASApp/blob/e39db4e8b5353bdf94406b3c048b9778b09745ee/ci/orion.sh#L4

**Comment by CoryMartin-NOAA** [comment]
hmm, okay, not sure why I thought that was the issue. Unless that is not being sourced/exported properly. I'll look more closely tomorrow.

**Comment by CoryMartin-NOAA** [comment]
Did the following on orion:
- cloned fresh GDASApp develop
- ./build.sh
- module use modulefiles
- module load GDAS/orion
- cd build
- `ctest -VV -R test_gdasapp_snow_apply_jediincr`

```
1/1 Test #1952: test_gdasapp_snow_apply_jediincr ...   Passed  230.82 sec

The following tests passed:
	test_gdasapp_snow_apply_jediincr

100% tests passed, 0 tests failed out of 1

Total Test time (real) = 231.57 sec
```

So I think the CI testing environment is not properly passing the Slurm variables...

**Comment by DavidNew-NOAA** [comment]
@CoryMartin-NOAA hmmm, I'll look into it

**Comment by DavidNew-NOAA** [comment]
Resolved by #1425
